sitewit/services.py

```diff
@@ -0,0 +1,68 @@
+import base64
+
+from demands import HTTPServiceClient
+from yoconfig import get_config
+
+
+class SitewitService(HTTPServiceClient):
+    """
+    Client for SiteWit's API::
+
+        sitewitservice = SitewitService()
+        response = sitewitservice.get_account(account_token).
+
+    """
+    DEFAULT_TIME_ZONE = 'Pacific Standard Time'
```

Comment:
It didn't work with the API. Once SiteWit fixes this problem, I'll switch to GMT.